packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -235,10 +254,13 @@ contract Deploy is Deployer {
     ////////////////////////////////////////////////////////////////
 
     /// @notice Deploy a full system with a new SuperchainConfig
-    ///         The Superchain system has 2 singleton contracts which lie outside of an OP Chain:
+    ///         The Superchain system has 3 singleton contracts which lie outside of an OP Chain:
     ///         1. The SuperchainConfig contract
     ///         2. The ProtocolVersions contract
-    function deploySuperchain() public {
+    ///         3. The SharedLockbox contract
+    function deploySuperchain(bool _isInterop) public {
```

Comment:
I dont fully understand why there is an argument here if its pulling from the deploy config anyways. I think what could be more useful is just pulling from the deploy config and then logging if `cfg.useInterop() == true`